src/main/src/main/java/org/geoserver/config/util/XStreamPersister.java

```diff
@@ -1506,10 +1511,49 @@ protected void postDoMarshal(Object source,
             }
         }
     }
+    protected class AbstractCatalogInfoConverter extends AbstractReflectionConverter {
```

Comment:
Go on give me a class javadoc, since this is protected it is part of the api and I would like to know what you are doing.